deploy/charts/emqx/templates/certificate.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if and (Values.ssl.enable) (not .Values.ssl.useExisting) -}}
```

Comment:
```suggestion
{{- if and (.Values.ssl.enable) (not .Values.ssl.useExisting) -}}
```

Reply:
Looks like it's my fault, please fix it

Reply:
Looks like it's my fault, please fix it

Reply:
could you please edit the commit log so that it can pass the git lint check

Reply:
> Could you please edit the commit log so that it can pass the git lint check

Not entirely sure where the problem is